ice.c

```diff
@@ -3929,6 +3942,9 @@ void *janus_ice_send_thread(void *data) {
 									stream->video_first_rtp_ts[0] = timestamp;
 								}
 							}
+							// update sent packets counter
```

Comment:
/* Update sent packets counter */

As a convention, we don't use `//` for comments, and sentences in comments always start capitalized.